<launch>
    <!-- Load parameters -->
    <rosparam file="$(find drone_autonomous)/config/hsv_params.yaml" command="load"/>

    <!-- Nodes -->
    <node pkg="drone_autonomous" type="camera_processor.py" name="camera_processor" output="screen"/>
    <node pkg="drone_autonomous" type="object_detector.py" name="object_detector" output="screen"/>
    <node pkg="drone_autonomous" type="drone_controller.py" name="drone_controller" output="screen"/>
    <node pkg="drone_autonomous" type="mission_manager.py" name="mission_manager" output="screen"/>
</launch>